packages/react/error-boundary/src/ErrorBoundary.en.md

```diff
@@ -31,3 +33,27 @@ It catches the error occurred on render or in `useEffect` callback and then rend
   <COMPONENT_CAN_CREATE_ERROR />
 </ErrorBoundary>
 ```
+
+## References
+
+- [jbee.io | Handling loading and error conditions declaratively](https://jbee.io/react/error-declarative-handling-1/)
+- [Slash 21 | Suspense and Error Handling](https://toss.im/slash-21/sessions/3-1)
+
+## useErrorBoundary
+
+React's [error boundary](https://reactjs.org/docs/error-boundaries.html) component cannot catch an error occurred in event handlers, asynchronous code like `setTimeout`, etc.
+
+`useErrorBoundary` is useful to deliver an error to the nearest error boundary in anywhere.
+
+```jsx
+const throwError = useErrorBoundary();
+
+<Button
+  onClick={() => {
+    if (someCondition) {
+      // Throws `new Error('throw error')` to the nearest `ErrorBoundary`.
+      throwError(new Error('throw error'));
+    }
+  }}
+/>;
+```
```

Comment:
I move document for useErrorBoundary from `useErrorBoundary page` to `ErrorBoundary page` in slash.page